tests/Console/Command/MoveEventsTest.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Console\Command;
+
+use Broadway\CommandHandling\Testing\TraceableCommandBus;
+use CultuurNet\UDB3\Event\Commands\UpdateLocation;
+use CultuurNet\UDB3\Event\ValueObjects\LocationId;
+use CultuurNet\UDB3\Model\ValueObject\Identity\ItemIdentifier;
+use CultuurNet\UDB3\Model\ValueObject\Identity\ItemIdentifiers;
+use CultuurNet\UDB3\Model\ValueObject\Identity\ItemType;
+use CultuurNet\UDB3\Model\ValueObject\Web\Url;
+use CultuurNet\UDB3\Search\Results;
+use CultuurNet\UDB3\Search\SearchServiceInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+use Symfony\Component\Console\Helper\HelperSet;
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\ArrayInput;
+use Symfony\Component\Console\Output\BufferedOutput;
+
+class MoveEventsTest extends TestCase
+{
+    private TraceableCommandBus $commandBus;
+
+    /**
+     * @var SearchServiceInterface|(SearchServiceInterface&object&MockObject)|(SearchServiceInterface&MockObject)|(object&MockObject)|MockObject
```

Comment:
Can't this just be `SearchServiceInterface&MockObject` or `SearchServiceInterface|MockObject`?